pallets/gear/src/tests.rs

```diff
@@ -1281,6 +1282,11 @@ fn read_state_bn_and_timestamp_works() {
         ));
 
         let program_id = utils::get_last_program_id();
+        assert_ok!(Gear::pay_rent(
```

Comment:
please, avoid such repetitions in tests by increasing free period for tests. to tests specific logic of rents, hack the stuff in tests directly by increasing current/start block number